Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mcr.microsoft.com/powershell:alpine-3.12
+
+ARG CLI_VERSION=latest
+
+LABEL name="m365pnp/cli-microsoft365:${CLI_VERSION}" \
+  description="Manage Microsoft 365 and SharePoint Framework projects on any platform" \
+  homepage="https://pnp.github.io/cli-microsoft365" \
+  maintainers="Waldek Mastykarz <waldek@mastykarz.nl>, \
+  Velin Georgiev <velin.georgiev@gmail.com>, \
+  Garry Trinder <garry.trinder@live.com>, \
+  Albert-Jan Schot <appie@digiwijs.nl>, \
+  Rabia Williams <rabiawilliams@gmail.com>, \
+  Mark Powney <mpowney@icloud.com>, \
+  Patrick Lamber <patrick@nubo.eu>" \
+  com.azure.dev.pipelines.agent.handler.node.path="/usr/bin/node"
+
+RUN apk add --no-cache \
+  curl \
+  sudo \
+  bash \
+  shadow \
+  bash-completion \
+  nodejs=12.18.4-r0 \
+  npm=12.18.4-r0
```

Comment:
Shouldn't this be npm@6?

Reply:
This version refers to `apk` package version of `npm` to pull which matches the `node` package, having checked the container it is running `v6.14.6`.

![image](https://user-images.githubusercontent.com/11563347/101662768-d245b100-3a41-11eb-9498-0fa1f5b214dd.png)

Reply:
Got it. Thanks for clarification